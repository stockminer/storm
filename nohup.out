[2015-09-16 10:20:03 29526 run-crawler.py:begin_work:27 ][INFO]: start thread info:spider_info:level-id[1-0],state[2]
[2015-09-16 10:20:03 29526 run-crawler.py:begin_work:27 ][INFO]: start thread info:spider_info:level-id[1-1],state[2]
[2015-09-16 10:20:03 29526 run-crawler.py:begin_work:27 ][INFO]: start thread info:spider_info:level-id[1-2],state[2]
[2015-09-16 10:20:03 29526 run-crawler.py:begin_work:27 ][INFO]: start thread info:spider_info:level-id[2-0],state[2]
[2015-09-16 10:20:03 29526 run-crawler.py:begin_work:27 ][INFO]: start thread info:spider_info:level-id[2-1],state[2]
[2015-09-16 10:20:03 29526 run-crawler.py:begin_work:27 ][INFO]: start thread info:spider_info:level-id[2-2],state[2]
[2015-09-16 10:20:03 29526 run-crawler.py:begin_work:27 ][INFO]: start thread info:spider_info:level-id[3-0],state[2]
[2015-09-16 10:20:03 29526 run-crawler.py:begin_work:27 ][INFO]: start thread info:spider_info:level-id[4-0],state[2]
[2015-09-16 10:20:03 29526 run-crawler.py:begin_work:27 ][INFO]: start thread info:spider_info:level-id[4-1],state[2]
[2015-09-16 10:20:03 29526 run-crawler.py:begin_work:27 ][INFO]: start thread info:spider_info:level-id[4-2],state[2]
[2015-09-16 10:20:03 29526 run-crawler.py:begin_work:27 ][INFO]: start thread info:spider_info:level-id[4-3],state[2]
[2015-09-16 10:20:03 29526 run-crawler.py:begin_work:27 ][INFO]: start thread info:spider_info:level-id[4-4],state[2]
[2015-09-16 10:20:03 29526 run-crawler.py:begin_work:27 ][INFO]: start thread info:spider_info:level-id[5-0],state[2]
[2015-09-16 10:20:03 29526 run-crawler.py:begin_work:27 ][INFO]: start thread info:spider_info:level-id[5-1],state[2]
[2015-09-16 10:20:03 29526 run-crawler.py:begin_work:27 ][INFO]: start thread info:spider_info:level-id[5-2],state[2]
[2015-09-16 10:20:03 29526 run-crawler.py:begin_work:27 ][INFO]: start thread info:spider_info:level-id[5-3],state[2]
[2015-09-16 10:20:03 29526 run-crawler.py:begin_work:27 ][INFO]: start thread info:spider_info:level-id[5-4],state[2]
[2015-09-16 10:20:03 29526 run-crawler.py:begin_work:27 ][INFO]: start thread info:spider_info:level-id[5-5],state[2]
