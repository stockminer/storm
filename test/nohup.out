python: can't open file 'run-crawler.py': [Errno 2] No such file or directory
